connection/mysql.go

```diff
@@ -680,6 +680,8 @@ func (dbw *DBWrapper) SqlFetchChecksums(table string, ids []string) (map[string]
 		if err != nil {
 			return nil, err
 		}
+
+		rows.Close()
```

Comment:
There already is a `defer rows.Close()`. Why the extra call?

Reply:
Because we're looping here, sry.

Reply:
To close the rows ASAP.